src/docs/07-operation-guide/02-maintain.md

```diff
@@ -69,3 +69,37 @@ Also, the schema tool by default will upgrade schema to the latest, so no manual
 
 Database schema changes are versioned in the folders: [Versioned Schema Changes](https://github.com/uber/cadence/tree/master/schema/mysql/v57/cadence/versioned) for Default Store
 and [Versioned Schema Changes](https://github.com/uber/cadence/tree/master/schema/mysql/v57/visibility/versioned) for Visibility Store if you use database for basic visibility instead of ElasticSearch.
+
+## Migrate Cadence cluster
+Migrating a Cadence cluster is rare, but could happen.
+There could be some reasons like:
+* Scale up -- move to a bigger cluster, with larger number of shards
```

Comment:
Is this true? I think our hashing will change if the number of shards changed. So we have to use the same number in two clusters.

Reply:
Thanks, I filed an issue and add a TODO in this sentense.